pkg/component/etcd/bootstrap.go

```diff
@@ -386,9 +386,8 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 	return managedresources.CreateForSeed(ctx, b.client, b.namespace, managedResourceControlName, false, resources)
 }
 
-func getDruidDeployCommands(etcdConfig *config.ETCDConfig) []string {
+func getDruidDeployArgs(etcdConfig *config.ETCDConfig) []string {
 	command := []string{
```

Comment:
`command` would rename that too to `args`

Reply:
dang